packages/wdio-appium-service/src/launcher.ts

```diff
@@ -182,30 +183,33 @@ export default class AppiumLauncher implements Services.ServiceInstance {
         log.warn(data.toString())
     }
 
-    onComplete() {
+    private promisifiedTreeKill = promisify<number, string>(treeKill)
+    async onComplete() {
         this._isShuttingDown = true
 
-        /**
-         * Kill appium and all process' spawned from it
-         */
         if (this._process && this._process.pid) {
-            /**
-             * remove stdio event listener
-             */
             this._process.stdout.off('data', this.#logStdout)
             this._process.stderr.off('data', this.#logStderr)
 
-            /**
-             * Ensure all child processes are also killed
-             */
```

Comment:
Why removing the comments?

Reply:
Not intentionally! Will put it back! Thanks